projects/rocprofiler-sdk/source/lib/python/rocpd/source/perfetto.cpp

```diff
@@ -177,27 +177,28 @@ write_perfetto(
     const tool::generator<types::memory_copies>&     memory_copy_gen,
     const tool::generator<types::scratch_memory>&    scratch_memory_gen,
     const tool::generator<types::memory_allocation>& memory_allocation_gen,
-    const tool::generator<types::counter>&           counter_collection_gen)
+    const tool::generator<types::counter>&           counter_collection_gen,
+    const tool::generator<types::pmc_event>&         pmc_event_gen)
 {
     namespace sdk    = ::rocprofiler::sdk;
     namespace common = ::rocprofiler::common;
 
-    static auto orig_process_track = ::perfetto::ProcessTrack::Current();
-    static auto orig_process_desc  = orig_process_track.Serialize();
+    static auto     orig_process_track = ::perfetto::ProcessTrack::Current();
+    static auto     orig_process_desc  = orig_process_track.Serialize();
+    static uint64_t global_flow_index  = 0;
+    static uint64_t global_track_index = 0;
```

Comment:
```suggestion
    static uint64_t global_flow_index  = 0x100000000;
```
I set it to some high value, just for the flow arrows

Reply:
Done.